tests/Doctrine/Tests/ORM/Functional/BasicFunctionalTest.php

```diff
@@ -727,7 +727,8 @@ public function testOneToOneNullUpdate()
      */
     public function testNewAssociatedEntityDuringFlushThrowsException()
     {
-        //$this->_em->getConnection()->getConfiguration()->setSQLLogger(new \Doctrine\DBAL\Logging\EchoSQLLogger);
+        $this->expectException(\InvalidArgumentException::class);
```

Comment:
Please put the `$this->expectException()` call *right before* the code block that throws